### PR TITLE
repair: improve performance and reliability

### PR DESCRIPTION
**Pull request by lheeger-jump**

### CODE REVIEW

File: src/app/fdctl/run/topos/fd_firedancer.c [hunk 1]
```diff
@@ -134,13 +134,13 @@ fd_topo_firedancer( config_t * _config ) {
...
   /**/                 fd_topob_link( topo, "repair_net",   "net_repair",   0,        config->tiles.net.send_buffer_size,       FD_NET_MTU,                    1UL   );
   /**/                 fd_topob_link( topo, "repair_sign",  "repair_sign",  0,        128UL,                                    2048UL,                        1UL );
   /**/                 fd_topob_link( topo, "sign_repair",  "sign_repair",  0,        128UL,                                    64UL,                          1UL );
   /**/                 fd_topob_link( topo, "store_replay", "store_replay", 0,        128UL,                                    FD_SHRED_MAX_PER_SLOT * FD_SHRED_MAX_SZ, 16UL  );
-  /**/                 fd_topob_link( topo, "replay_poh",   "replay_poh",   0,        1024UL,                                   128UL*1024UL*1024UL,           16UL  );
+  /**/                 fd_topob_link( topo, "replay_poh",   "replay_poh",   0,        1024UL,                                   sizeof(ulong)+sizeof(fd_hash_t)+(1024UL*sizeof(fd_txn_p_t)),           16UL  );
```
Comment (arjain4) [inline src/app/fdctl/run/topos/fd_firedancer.c:143]: store -> replay is this format
replay ->poh is actually 
`1024 * sizeof(fd_txn_p_t) + sizeof(fd_microblock_trailer_t)`

**Review by arjain4 — CHANGES_REQUESTED** [review]

File: src/app/fdctl/run/topos/fd_firedancer.c (hunk 1 above)
Comment (lheeger-jump) [reply src/app/fdctl/run/topos/fd_firedancer.c:143]: rip

Comment (lheeger-jump) [reply src/app/fdctl/run/topos/fd_firedancer.c]: fixed

File: src/app/fdctl/run/topos/fd_firedancer.c [hunk 2]
```diff
@@ -134,13 +134,13 @@ fd_topo_firedancer( config_t * _config ) {
...
 
-  /**/                 fd_topob_link( topo, "store_repair", "store_repair", 0,        128UL,                                    64UL * 32768UL,                16UL  );
+  /**/                 fd_topob_link( topo, "store_repair", "store_repair", 0,        1024UL,                                   USHORT_MAX,                    16UL  );
   /**/                 fd_topob_link( topo, "repair_store", "repair_store", 0,        1024UL*1024UL,                            FD_SHRED_MAX_SZ,               128UL );
   /**/                 fd_topob_link( topo, "repair_net",   "net_repair",   0,        config->tiles.net.send_buffer_size,       FD_NET_MTU,                    1UL   );
   /**/                 fd_topob_link( topo, "repair_sign",  "repair_sign",  0,        128UL,                                    2048UL,                        1UL );
   /**/                 fd_topob_link( topo, "sign_repair",  "sign_repair",  0,        128UL,                                    64UL,                          1UL );
   /**/                 fd_topob_link( topo, "store_replay", "store_replay", 0,        128UL,                                    FD_SHRED_MAX_PER_SLOT * FD_SHRED_MAX_SZ, 16UL  );
```
Comment (arjain4) [inline src/app/fdctl/run/topos/fd_firedancer.c:142]: Should this be changed to what can be sent from store?

Comment (lheeger-jump) [reply src/app/fdctl/run/topos/fd_firedancer.c:142]: Yes, but not right now.

**Review by arjain4 — APPROVED** [review]

**Pull request merged**